rc/filetype/css.kak

```diff
@@ -32,28 +33,78 @@ provide-module css %[
+  # html tags
+  # generated from the URL & <code> below
+  # includes elements that cannot be styled, which is fine.
+  # 
+  # https://developer.mozilla.org/en-US/docs/Web/HTML/Element
+  html_tags='html body address article aside footer header h1 h2 h3 h4 h5 h6 main nav section blockquote dd div dl dt figcaption figure hr li ol p pre ul a abbr b bdi bdo br cite code data dfn em i kbd mark q rp rt ruby s samp small span strong sub sup time u var wbr area audio img map track video embed iframe object param picture portal source canvas noscript script del ins caption col colgroup table tbody td tfoot th thead tr button datalist fieldset form input label legend meter optgroup option output progress select textarea details dialog menu summary slot template acronym applet basefont bgsound big blink center content dir font frame frameset hgroup image keygen marquee menuitem nobr noembed noframes plaintext rb rtc shadow spacer strike tt xmp'
+
+  # Units
+  # ‾‾‾‾‾
+  # generated from the URL & <code> below
+  # includes #rgb, #rrggbb, #rrggbbaa as color values {3,8}
+  # 
+  # https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units
+  units='% cap ch cm deg dpcm dpi dppx em ex grad Hz ic in kHz lh mm ms pc pt px Q rad rem rlh s turn vb vh vi vmax vmin vw x'
+
+  logical_ops='and not only from to'
+  keywords='!important auto inherit initial unset none'
+  media_types='all print screen speech'
+
+  # easing_re='linear|ease(-(in-out|in|out))?|step-(start|end)'
+
+  # combinators='+ > ~ ||'
+  # attribute_op='= ~= |= ^= $= *='
+
+  join() { eval set -- $1; IFS="|"; echo "$*"; }
+
+  # Selectors
+  # ‾‾‾‾‾‾‾‾‾
+  # universal: *, ns|*, *|*, |*
+  # class/id: .class, #id
+  # type: element
+  # attr: [attr=val]
+
+  # order below matters
+  printf %s "
+  add-highlighter shared/css/code/tag_selectors regex \b($(join "${html_tags}"))((:[a-z:])|[\h.#]) 1:keyword
 
-add-highlighter shared/css/declaration/base default-region group
-add-highlighter shared/css/declaration/double_string region '"' (?<!\\)(\\\\)*" fill string
-add-highlighter shared/css/declaration/single_string region "'" "'"             fill string
-add-highlighter shared/css/declaration/comment region /[*] [*]/ fill comment
+  add-highlighter shared/css/code/functional_notation regex ([a-zA-Z0-9-_]+[a-zA-Z0-9])\( 1:keyword
 
-# https://developer.mozilla.org/en-US/docs/Web/CSS/length
-add-highlighter shared/css/declaration/base/ regex (#[0-9A-Fa-f]+)|((\d*\.)?\d+(ch|cm|em|ex|mm|pc|pt|px|rem|vh|vmax|vmin|vw)) 0:value
+  add-highlighter shared/css/code/logical_operators regex (\b($(join "${logical_ops}"))\b|$(join "${keywords}")) 1:keyword 1:+i
 
-add-highlighter shared/css/declaration/base/ regex ([A-Za-z][A-Za-z0-9_-]*)\h*: 1:keyword
-add-highlighter shared/css/declaration/base/ regex :(before|after) 0:attribute
-add-highlighter shared/css/declaration/base/ regex !important 0:keyword
+  add-highlighter shared/css/code/media_types regex \b($(join "${media_types}"))\b 1:+i
```

Comment:
As above, this should be a standard face, or a custom face that defaults to a standard face (possibly with some alterations)

Reply:
Again, using simply italics here and following vscode styles mostly -- with default face. Not sure what the issue or concern is.